veniq/dataset_collection/types_identifier.py

```diff
@@ -325,8 +325,7 @@ def get_lines_of_method_body(
         line_with_declaration = lines[invocation_line - 1].split('=')
         is_var_declaration = self.is_var_declaration(lines, invocation_line)
         is_direct_return = self.is_direct_return(lines, invocation_line)
-        spaces_in_body = self.complement_spaces(body_start_line + 1, invocation_line, lines) * ' '
-
+        spaces_in_body = self.complement_spaces(body_start_line, invocation_line, lines) * ' '
```

Comment:
We better multiply string by integer, not vice versa.

Reply:
why is that?

Reply:
Because it is not that obvious what is happening here. If we will see string first, we will understand quicker that  result is going to be another string. For now, one can consider that this expression is integer.

Reply:
fixed